### PR TITLE
Create Frontend Documentation

### PR DESCRIPTION
**Pull request by mhellmeier**
This pull request consists of an initial version of the frontend documentation.

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
\ No newline at end of file
+# PURIS Frontend Documentation
+
+## Frontend Installation
+
+### Requirements
+To use the PURIS frontend, you need:
+
+- a running [PURIS backend](https://github.com/eclipse-tractusx/puris-backend/)
+- a running [Product EDC](https://github.com/catenax-ng/product-edc)
```
Comment (carslen) [inline docs/documentation.md:9]: Please do not reference catenax-ng repositories. EDC is also available in [Eclipse Tractusx](https://github.com/eclipse-tractusx/tractusx-edc).

File: docs/documentation.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+- a running [CX backend service](https://github.com/denisneuling/cx-backend-service)
```
Comment (carslen) [inline docs/documentation.md:10]: Pretty unsure if this is ok @Siegfriedk @SebastianBezold @almadigabor. See also Line 69 in this file.

**Review by carslen — CHANGES_REQUESTED** [review]
Please refer to comments.

File: docs/documentation.md (hunk 2 above)
Comment (almadigabor) [reply docs/documentation.md:10]: I also don't understand the intent of this cx-backend-service, checked the repo, Dockerfile, chart but I think it shouldn't be referenced in an eclipse-tractusx repo.

Comment (Siegfriedk) [reply docs/documentation.md:10]: I think you can reference it but the implication of it makes it potentially an issue: If PURIS requires NON eclipse foundation project (espeically the denisneuling one), you are not contributing all necessary product components of PURIS to eclpse foundation.

Comment (mhellmeier) [reply docs/documentation.md:10]: The PURIS application uses the EDC (to be more concrete, the [product-edc](https://github.com/catenax-ng/product-edc) / [tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc)).

@almadigabor : You mentioned that:
> [...] I think it shouldn't be referenced in an eclipse-tractusx repo.

Which is in line with @Siegfriedk s comment:
> If PURIS requires NON eclipse foundation project (espeically the denisneuling one), you are not contributing all necessary product components of PURIS to eclpse foundation.

As mentioned, we are using the EDC. And the EDC itself in the latest version in the Eclipse Tractus-X space uses components not part of the eclipse foundation.
See:

- https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/src/main/resources/deployment/helm/all-in-one/Chart.yaml#L48
- https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/src/main/resources/deployment/helm/all-in-one/Chart.yaml#L75

Therefore, the problem isn't related to this PURIS application directly but more or less an upstream problem in the tractusx-edc.

File: docs/documentation.md (hunk 1 above)
Comment (mhellmeier) [reply docs/documentation.md:9]: We will update this in a newer version!

File: docs/documentation.md (hunk 2 above)
Comment (mhellmeier) [reply docs/documentation.md:10]: Since the "CX backend service", we will update this in a newer version!

**Comment by mhellmeier** [comment]
Due to different changes in the development, updated frontend documentation will be created in a new format and a new version.

**Pull request closed without merging**